### PR TITLE
youtube-dl-GUI for Python 3 using PyQt5

### PR DESCRIPTION
**Pull request by oleksis**
Addd ThreadPool
More stable

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,18 @@
...
 ![youtube-dl-gui Screenshot in Ubuntu](http://imgur.com/KxTLeYl.png)
 
 Requirements:
 ------------
-- PyQt4
-- [youtube_dl](https://github.com/rg3/youtube-dl)
-- Python >= v2.6
-- py2exe - _see Windows_
+- PyQt5
+- [youtube_dl](https://gitlab.com/dstftw/youtube-dl)
```
Comment (yasoob) [inline README.md]: youtube-dl is back on GitHub. Can you change the url to https://github.com/ytdl-org/youtube-dl everywhere?

File: README.md [hunk 2]
```diff
@@ -25,38 +23,37 @@ This application has the following features:
...
 ---------
 - Muhammad Yasoob Ullah Khalid
+- Oleksis Fraga <oleksis.fraga@gmail.com>
 
 If you want to become a contributor then just contribute some code and I will add you to this list
 
 Bugs:
 ----------
-This program is still in beta so if you encounter any bugs feel free to report them on https://github.com/yasoob/youtube-dl-GUI/issues.
+This program is still in beta so if you encounter any bugs feel free to report them on https://github.com/oleksis/youtube-dl-pyqt/issues.
```
Comment (yasoob) [inline README.md]: Let's retain the current issues page. I can add you as a contributor to the project if you want. :)

Comment (oleksis) [reply README.md]: I make the changes. I would like to keep the project updated and be a collaborator (maintainer) 🥳

**Comment by yasoob** [comment]
I have invited you as a collaborator. I did a cursory review of the PR and it looks good but you can go ahead and finish it up if it looks good to you. You should have access to merge it now. I don't have enough bandwidth to explore it in detail. 

I hope the project grows under your guidance :)

**Comment by oleksis** [comment]
I am going to join this PR which is the version compatible with PyQt5 in Python3. I am going to review the Conversion part and update youtube-dl. Thanks for adding me to the project!

**Pull request merged**